index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+      <div class="menu">
+        <a href="#profile-link">About</a>
+        <a>Work</a>
+        <a>Contact</a>
+      </div>
+    </nav>
+    <section id="welcome-section">
+      <div class="div-welcome">
+        <h1>Hey, I'm Chris and David</h1>
+      </div>
+      </section>
```

Comment:
Almost perfect!!😮
Aligning opening and closing tags helps others read and understand your code better. In this case the opening and closing "section" tags aren't align and it might make it a little confusing to read.